docs/generated/packages/expo/executors/update.json

```diff
@@ -25,7 +25,7 @@
       },
       "republish": {
         "type": "boolean",
-        "description": "Enable JSON output, non-JSON messages will be printed to stderr",
+        "description": "Republish a previous update within a branch",
```

Comment:
this is the generated doc. i think what you meant to change is in file packages/expo/src/executors/update/schema.json

Reply:
@xiongemi thank you, i've corrected that